indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -157,7 +165,18 @@ function EditableListDisplay({
         options: [
           {value: 'not_submitted', text: Translate.string('Not submitted'), color: 'default'},
           {value: 'ready_for_review', text: Translate.string('Ready for review'), color: 'grey'},
-          {value: 'accepted', text: Translate.string('Accepted'), exclusive: true, color: 'green'},
+          {
+            value: 'accepted',
+            text: Translate.string('Accepted by editor'),
+            exclusive: true,
+            color: 'green',
+          },
+          {
+            value: 'accepted_submitter',
+            text: Translate.string('Accepted by submitter'),
+            exclusive: true,
+            color: 'green',
```

Comment:
Maybe a slightly different color would look nicer here.. SUI has this `olive` color that looks quite nice:
![image](https://github.com/indico/indico/assets/27357203/cb4f1eb8-fc98-44df-9c06-b016072864c3)